MyArrayList/Program.cs

```diff
@@ -122,11 +136,13 @@ public void ReverseList()
         }
         _list = array;
     }
+
     //returns value of some index
     public int GetElement(int index)
     {
         return _list[index];
```

Comment:
What going to happen if I try to get element by index 9 from array = { 1, 2, 3 } ?